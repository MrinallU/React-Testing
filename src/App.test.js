import React from 'react';
import App from './App';
import ReactDOM from 'react-dom';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';   // Import the nessecary stuff

Enzyme.configure({ adapter: new Adapter() }) // Creates the text

it('renders correctly enzyme', () => {
 const wrapper = shallow(<App />)
 expect(toJson(wrapper)).toMatchSnapshot();  
});

// To update test after testing press "w" and then "u"
