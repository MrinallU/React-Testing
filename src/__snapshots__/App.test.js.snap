// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <Button />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
  </header>
</div>
`;
